core/modules/utils/dom/sniff-chrome-touch.js

```diff
@@ -0,0 +1,38 @@
+/*\
+title: $:/core/modules/utils/dom/sniff-chrome-touch.js
+type: application/javascript
+module-type: utils
+
+Utility functions to detect chrome-like browsers and touch input
```

Comment:
We already have $tw.browser.* flags being applied in `$:/core/modules/startup.js`, I don't like the idea of a separate set of methods elsewhere.

I'd prefer to extend the definitions in `$:/core/modules/startup.js` to include these three new ones. It would be nice to move the $tw.browser.* stuff out into a specific module type but we can leave that for another time.

Reply:
OK I did update it as requested.